### PR TITLE
[catalog-server] Handle invalid and not found packages better.

### PR DESCRIPTION
**Pull request by justinfagnani**
This handles a few cases better, and tests that the db state is what we expect:
1. Not found packages
2. Packages with no customElements field in package.json
3. Packages with a customElements field in package.json, but a missing file (something we see in the bootstrap packages)

### CODE REVIEW

File: packages/catalog-server/data/bootstrap-packages.json [hunk 1]
```diff
@@ -1,40 +1,5 @@
 {
   "packages": [
-    "@shoelace-style/shoelace",
```
Comment (aomarks) [inline packages/catalog-server/data/bootstrap-packages.json:3]: Intentionally removed most of the bootstrap components?

File: packages/catalog-server/package.json [hunk 2]
```diff
@@ -56,7 +56,7 @@
       ]
     },
     "test": {
-      "command": "NODE_OPTIONS=--enable-source-maps FIRESTORE_EMULATOR_HOST=localhost:7450 uvu test \"_test\\.js$\"",
+      "command": "NODE_OPTIONS=--enable-source-maps FIRESTORE_EMULATOR_HOST=localhost:7450 uvu test \"catalog_test\\.js$\"",
```
Comment (aomarks) [inline packages/catalog-server/package.json:59]: Intentionally scoping down tests to just catalog ones?

File: packages/catalog-server/src/lib/catalog.ts [hunk 3]
```diff
@@ -100,20 +103,31 @@ export class Catalog {
...
     }
-    console.log(' done');
 
     if (newPackage === undefined) {
-      await this.#repository.endPackageImportWithNotFound(packageName);
-      // TODO (justinfagnani): a crazy edge case would be a package that was
-      // previously found, but is not found now. Update package versions?
-      return {};
+      throw new Error(`Internal error importing package ${packageName}`);
+      // await this.#repository.endPackageImportWithNotFound(packageName);
```
Comment (aomarks) [inline packages/catalog-server/src/lib/catalog.ts:127]: Remove commented out code?

**Review by aomarks — APPROVED** [review]

File: packages/catalog-server/package.json (hunk 2 above)
Comment (justinfagnani) [reply packages/catalog-server/package.json:59]: oops, I didn't mean to check that in

File: packages/catalog-server/data/bootstrap-packages.json (hunk 1 above)
Comment (justinfagnani) [reply packages/catalog-server/data/bootstrap-packages.json:3]: nope, oops!

**Pull request merged**